### ISSUE TITLE
The demo is not visible in the elements catalog

### ISSUE THREAD

**Issue by arthurevans** [PolymerElements/gold-cc-cvc-input#38]
_From @notwaldorf on January 29, 2016 23:16_

_From @zoechi on November 27, 2015 7:17_

https://elements.polymer-project.org/elements/gold-cc-cvc-input

_Copied from original issue: PolymerElements/gold-cc-cvc-input#33_

_Copied from original issue: Polymer/polymer-element-catalog#243_

**Comment by arthurevans** [comment]
_From @notwaldorf on January 29, 2016 23:16_

Ah, good catch. Il'll take a look. Here's what the demo should point to, in the meantime: https://elements.polymer-project.org/elements/gold-cc-cvc-input?view=demo:demo/index.html&active=gold-cc-cvc-input

**Comment by arthurevans** [comment]
_From @notwaldorf on January 29, 2016 23:16_

Going to move this to the polymer-elements-catalog, since this element seems to be doing everything fine (it has an annotation, the `bower.json` looks right)

**Comment by arthurevans** [comment]
So I looked at this and noticed that this element has an (obsolete) `@class` tag, and that it occurs before the `@demo` tag.

Going on a hunch, I tried removing it. Voila, demo! The same tag also occurs in the other gold elements and in `iron-pages`, but in those cases it's _after_ the `@demo` tag (or in one case, I think, above but not immediately above `@demo`)... Flipping the order of the tags in `gold-zip-input.html`, I produced the same problem (no demo showing up in the catalog). So I think that's your culprit.

The good news is, it doesn't do anything, so we should just delete it from all of our elements.
